geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSDiffIntegrationTest.java

```diff
@@ -167,6 +167,30 @@ public void sdiffWithDifferentyKeyType_returnsWrongTypeError() {
     assertThatThrownBy(() -> jedis.sdiff("ding")).hasMessageContaining(ERROR_WRONG_TYPE);
   }
 
+  @Test
+  public void sdiff_withDifferentKeyTypeAndTwoSetKeys_returnsWrongTypeError() {
+    String diffTypeKey = "{tag1}ding";
```

Comment:
Could this (and the other variables with the same name added in this PR) be renamed to `stringKey` to avoid confusion around "diff" meaning different vs the DIFF operation?